tests/conftest.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+
+from arcus import Client
+
+
+@pytest.fixture(scope='module')
+def vcr_config():
+    config = dict()
+    # Replace the Authorization request header with 'DUMMY' in cassettes
+    config['filter_headers'] = [('Authorization:', 'DUMMY')]
```

Comment:
None of the auth headers were replaced. Did you add this code *after* creating the cassettes?

Regardless, it's actually unnecessary since the auth header has a hash—instead of the actual auth tokens. I'd recommend removing this entirely for this particular instance.

Reply:
corregido